one_fm/one_fm/custom/shift_assignment.json

```diff
@@ -139,41 +143,42 @@
    "collapsible": 0,
    "collapsible_depends_on": null,
    "columns": 0,
-   "creation": "2020-10-06 13:39:37.549249",
+   "creation": "2024-03-25 01:19:32.655200",
    "default": null,
    "depends_on": null,
    "description": null,
    "docstatus": 0,
    "dt": "Shift Assignment",
-   "fetch_from": "shift.project",
+   "fetch_from": null,
    "fetch_if_empty": 0,
-   "fieldname": "project",
+   "fieldname": "replaced_shift_assignment",
    "fieldtype": "Link",
    "hidden": 0,
    "hide_border": 0,
    "hide_days": 0,
    "hide_seconds": 0,
-   "idx": 5,
+   "idx": 12,
    "ignore_user_permissions": 0,
    "ignore_xss_filter": 0,
    "in_global_search": 0,
    "in_list_view": 0,
    "in_preview": 0,
    "in_standard_filter": 0,
-   "insert_after": "site",
-   "is_system_generated": 1,
+   "insert_after": "is_replaced",
+   "is_system_generated": 0,
    "is_virtual": 0,
-   "label": "Project",
+   "label": "Replaced Shift Assignment",
    "length": 0,
+   "link_filters": null,
    "mandatory_depends_on": null,
-   "modified": "2020-10-06 13:39:37.549249",
-   "modified_by": "Administrator",
+   "modified": "2024-03-25 01:19:32.655200",
+   "modified_by": "s.shaikh@one-fm.com",
    "module": null,
-   "name": "Shift Assignment-project",
+   "name": "Shift Assignment-replaced_shift_assignment",
    "no_copy": 0,
    "non_negative": 0,
-   "options": "Project",
-   "owner": "Administrator",
+   "options": "Shift Assignment",
+   "owner": "s.shaikh@one-fm.com",
```

Comment:
Replace the personal email id with `Administrator`
Be careful while editing the json file. After edit run a bench migrate on your local